tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -203,6 +203,23 @@ public void TestIndexer()
             Assert.Throws<ArgumentException>(() => dataFrame["Int5"]);
         }
 
+        [Fact]
+        public void GetColumnTests()
+        {
+            DataFrame dataFrame = MakeDataFrameWithAllColumnTypes(10);
+            PrimitiveDataFrameColumn<int> ints = dataFrame.Columns.GetPrimitiveColumn<int>("Int");
+            Assert.NotNull(ints);
+            Assert.Throws<ArgumentException>(() => dataFrame.Columns.GetPrimitiveColumn<float>("Int"));
+
+            StringDataFrameColumn strings = dataFrame.Columns.GetStringColumn("String");
+            Assert.NotNull(strings);
+            Assert.Throws<ArgumentException>(() => dataFrame.Columns.GetStringColumn("String"));
+
+            ArrowStringDataFrameColumn arrowStrings = dataFrame.Columns.GetArrowStringColumn("ArrowString");
+            Assert.NotNull(arrowStrings);
+            Assert.Throws<ArgumentException>(() => dataFrame.Columns.GetArrowStringColumn("ArrowString"));
```

Comment:
Why are you expecting this to be NotNull above, but expecting it to throw here?

Reply:
That was a copy-paste error. Fixed